builtin/stash--helper.c

```diff
@@ -689,6 +695,50 @@ static int show_stash(int argc, const char **argv, const char *prefix)
 	return ret;
 }
 
+static int do_store_stash(const char *stash_msg, const char *w_commit)
+{
+	struct object_id obj;
+	int ret = 0;
+
+	ret = get_oid(w_commit, &obj);
+	if (!ret) {
+		ret = update_ref(stash_msg, ref_stash, &obj, NULL,
+			REF_FORCE_CREATE_REFLOG, 0);
+	}
+
+	if (ret && !quiet)
+		fprintf_ln(stderr, _("Cannot update %s with %s"), ref_stash, w_commit);
+
+	return ret;
+}
+
+static int store_stash(int argc, const char **argv, const char *prefix)
+{
+	const char *stash_msg = NULL;
+	const char *w_commit = NULL;
+	struct option options[] = {
+		OPT__QUIET(&quiet, N_("be quiet, only report errors")),
+		OPT_STRING('m', "message", &stash_msg, "message", N_("stash message")),
+		OPT_END()
+	};
+
+	argc = parse_options(argc, argv, prefix, options,
+		git_stash_helper_store_usage, PARSE_OPT_KEEP_UNKNOWN);
+
+	if (argc == 0) {
+		fprintf(stderr, "\"git stash--helper\" requires one <commit> argument\n");
```

Comment:
I see this string is currently translated, which I think it should continue to be.  Just wrap the string with `_(<string>)` to mark it for translation.  

Also this is missing `store` after `git stash--helper`.

I also wonder if we should just keep this as `git stash` instead of `git stash--helper`, as `stash--helper` feels more like an implementation detail of `git stash` that we're using during the rewrite, but will the go away I assume.

Reply:
We just need to make sure that `git stash store` hands off to `stash--helper`, then the end user will never need to know that `stash--helper` even exists, I agree!